### PR TITLE
Add Driving innovation, Who we are?, Services and Team sections

### PR DESCRIPTION
**Pull request by Grappler2010**

### CODE REVIEW

File: app/components/ServicesSection/ServicesSection.tsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  return (
+    <MainContainer>
+      <section className={styles.component}>
+        <H2 className={styles.title}>Services</H2>
+        <P className={styles.servicesTitle}>Our services</P>
+        <div className={styles.servicesContainer}>
+          <Image className={styles.image} src={ourServicesSrc} alt="OurServices" />
+          <div className={styles.cardsContainer}>
+            <div className={styles.servicesCard}>
+              <P className={styles.number}>01.</P>
```
Comment (dealwith) [inline app/components/ServicesSection/ServicesSection.tsx:18]: should be semantically `ul > li`, I guess

File: app/components/DrivingInnovationSection/DrivingInnovationSection.tsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+import styles from "./drivingInnovationSection.module.scss";
+import imageSrc from "./img/innovationImage.jpg";
+
+export const DrivingInnovationSection = () => {
+  return (
+    <MainContainer>
+      <section className={styles.component}>
+        <H2 className={styles.title}>Driving innovation and growth in finance</H2>
+        <div className={styles.contentWrapper}>
+          <Image className={styles.image} src={imageSrc} alt="Image" />
```
Comment (dealwith) [inline app/components/DrivingInnovationSection/DrivingInnovationSection.tsx:13]: These images shouldn't come from src, they should come from contentful CMS, please add images to Contentful

**Review by dealwith — CHANGES_REQUESTED** [review]
The PR is good, but you didn't integrate the CMS, please integrate it first

**Review by dealwith — APPROVED** [review]

**Comment by dealwith** [comment]
Forget about CMS comment above, I will first discuss it with Julian

File: app/components/ServicesSection/ServicesSection.tsx (hunk 1 above)
Comment (Grappler2010) [reply app/components/ServicesSection/ServicesSection.tsx:18]: Maybe we will change them to components in the future

**Pull request merged**